src/naive/date/mod.rs

```diff
@@ -1195,58 +1195,51 @@ impl NaiveDate {
 
     /// Makes a new `NaiveDate` with the year number changed, while keeping the same month and day.
     ///
-    /// This method assumes you want to work on the date as a year-month-day value. Don't use it if
-    /// you want the ordinal to stay the same after changing the year, of if you want the week and
-    /// weekday values to stay the same.
```

Comment:
I moved this documentation from the `Datelike` trait to here in #1476. Did you remove it on purpose or is it a rebase thing?

Reply:
Rebase artifact... will fix.